UniConnectApp/app/src/main/java/com/example/uniconnect/MainActivity.kt

```diff
@@ -57,28 +59,76 @@ class MainActivity : ComponentActivity() {
         var description by remember { mutableStateOf("")}
         //var postID by remember { mutableStateOf("")}
         val context = LocalContext.current
-        Column{
-            OutlinedTextField(
-                value = title,
-                onValueChange = {title = it},
-                label = { Text(stringResource(R.string.title))}
+        Column(
+            Modifier.fillMaxHeight(),
+            verticalArrangement = Arrangement.SpaceBetween
+        ) {
+            Text(
+                text = stringResource(id = R.string.post_title),
+                style = MaterialTheme.typography.h6,
+                modifier = Modifier.padding(start = 20.dp)
             )
-            OutlinedTextField(
-                value = description,
-                onValueChange = {description =it},
-                label = { Text(stringResource(R.string.description))}
-            )
-            Button(
-                onClick = {
-                    var post = Post(title = title, description = description)
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(130.dp)
+                    .padding(horizontal = 30.dp),
+                horizontalArrangement = Arrangement.Center,
+            ) {
+                OutlinedTextField(
+                    value = title,
+                    onValueChange = {title = it},
+                    label = { Text(stringResource(R.string.title))},
+                    shape = RoundedCornerShape(20.dp),
+                    modifier = Modifier
+
+                        .fillMaxWidth()
+                )
+            }
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(500.dp)
+                    .padding(horizontal = 30.dp),
+                horizontalArrangement = Arrangement.Center,
+            ) {
+                OutlinedTextField(
+                    value = description,
+                    onValueChange = {description = it},
+                    label = { Text(stringResource(R.string.description))},
+                    shape = RoundedCornerShape(20.dp),
+                    modifier = Modifier
+                        .fillMaxSize()
+                )
+            }
+            Row(
+                horizontalArrangement = Arrangement.SpaceBetween,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = 30.dp, horizontal = 30.dp)
+            ){
```

Comment:
Placed the buttons within a row that lies toward the bottom of the screen, rather than being on top of each other.